autonat/autonat-v2.md

```diff
@@ -3,7 +3,7 @@
 
 | Lifecycle Stage | Maturity                 | Status | Latest Revision |
 |-----------------|--------------------------|--------|-----------------|
-| 1A              | Working Draft            | Active | r1, 2023-04-12  |
+| 1A              | Working Draft            | Active | r2, 2023-04-15  |
```

Comment:
No need to bump the revision for unmerged specs.

Reply:
ah thanks!